eve/io/mongo/__init__.py

```diff
@@ -10,7 +10,7 @@
     :license: BSD, see LICENSE for more details.
 """
 
+from eve.io.mongo.media import GridFSMediaStorage
```

Comment:
tests are failing because of this line. If you move it back to the bottom, they'll pass (imports order is important in this case.)